lookout/style/format/code_generator.py

```diff
@@ -57,6 +57,43 @@ def __init__(self,
         self.url = url
         self.commit = commit
 
+    def generate_new_line(self, line_vnodes: List[VirtualNode]) -> str:
+        """
+        Generate new code line for giving vnodes.
+
+        :param line_vnodes: corresponding virtual nodes sequence. It should be started from token \
+                            with indentation change or newline token from the prevoius line if \
+                            exists. The final line break of the line should not be included.
+        :return: Code line.
+        """
+        if not line_vnodes:
+            return ""
+
+        quotes_indx = {CLASS_INDEX[CLS_DOUBLE_QUOTE], CLASS_INDEX[CLS_SINGLE_QUOTE]}
```

Comment:
quotes are a huge problem here. because you can have `"⏎␣⁺␣⁺` and it can be changed to `'⏎⏎␣⁺␣⁺␣⁺␣⁺`. So you should handle them with crazy code.
I take care only about a second part: newlines and indentation. 

In this PR we miss quotes at the end of the line if there are any suggestion from style-analyzer. 
So the code will be even more ad-hoc. I think it is a good time to think about separation quotes from another style tokens as part of https://github.com/src-d/style-analyzer/issues/525

Reply:
@zurk Can you please put the first paragraph as a comment to this code

Reply:
I simplify this part of the code. Now dealing with quotes is better.